v2/loader/schema_parser_source.go

```diff
@@ -58,13 +59,16 @@ func NewSchemaParserSource(fpath string) (SchemaSource, error) {
 			v := tables[tableName]
 			v.createIndexes = append(v.createIndexes, val)
 			tables[tableName] = v
+		case *spansql.CreateChangeStream:
+			log.Printf("skipped. CreateChangeStream isn't supported yet. got ' %s'", ddlstmt.SQL())
```

Comment:
Please remove this line. (If we do this, it's better to implement warning and it should output to stderr)